package.json

```diff
@@ -8,8 +8,7 @@
     "compile": "tsc --build tsconfig.build.json",
     "compile:clean": "npm run compile -- --clean",
     "watch": "npm run compile && tsc --build tsconfig.json --watch",
-    "version": "npm i --package-lock-only && node ./prompt-update-changelogs.js",
-    "release:version-bump": "lerna version",
```

Comment:
This might seem redundant or extraneous but generally speaking I think npm scripts tell a lot of things about how to work within any particular repo. Having this here is a form of documentation that I consider to be valuable.

Reply:
Good point, I'll add it back.

For context, I initially removed this because I ran this command and hit `ctrl-c` when being prompted to commit changelogs and it still finished the release. It was a footgun that I wanted to remove, but I believe it might've been my misunderstanding how the `prompt-update-changelogs.js` script worked (and maybe that script would need to account for `ctrl-c`).

Reply:
Oh, this comment was directed at the `release:version-bump` script. I think it's easy to say "yeah you just run `npx lerna version`" but it's nice to have that written out plainly in the scripts which to me says "this is a regular thing we do often".

Reply:
Added back `release:version-bump`